cleo/io/buffered_io.py

```diff
@@ -19,32 +20,29 @@ def __init__(
         supports_utf8: bool = True,
     ) -> None:
         super().__init__(
-            input,
+            input or StringInput(""),
             BufferedOutput(decorated=decorated, supports_utf8=supports_utf8),
             BufferedOutput(decorated=decorated, supports_utf8=supports_utf8),
         )
 
-        self._output = cast(BufferedOutput, self._output)
```

Comment:
😢 dang, seems this is good enough for pyright

Reply:
Yeah, this is a pain. My guess is that `BufferedOutput` is different enough for `mypy` to consider it not deriving from `Output`.